src/components/ServiceDetails/AboutService/ServiceDemo/ThirdPartyAIService.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { withStyles } from "@material-ui/styles";
+
+import SampleServices from "../../../../assets/thirdPartyServices";
+import { useStyles } from "./styles";
+import { serviceActions } from "../../../../Redux/actionCreators";
+import { APIEndpoints } from "../../../../config/APIEndpoints";
+import Session from "../../../../utility/constants/Session";
```

Comment:
`Session` could move out of constants. Could be more closer to store.

Reply:
[done](https://github.com/singnet/refactored-dapp/pull/13/commits/bb0296bb37189c4bf226d4d7fdccf7b13ca205ab)